resources/athena-work-groups.go

```diff
@@ -0,0 +1,168 @@
+package resources
+
+import (
+	"errors"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/athena"
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/rebuy-de/aws-nuke/pkg/types"
+	"github.com/sirupsen/logrus"
+)
+
+func init() {
+	register("AthenaWorkGroup", ListAthenaWorkGroups)
+}
+
+type AthenaWorkGroup struct {
+	svc  *athena.Athena
+	name *string
+	arn  *string
+}
+
+func ListAthenaWorkGroups(sess *session.Session) ([]Resource, error) {
+	svc := athena.New(sess)
+	resources := []Resource{}
+
+	// Lookup current account ID
```

Comment:
This is a little painful, but unfortunately none of the Athena WorkGroup APIs return the full ARN of the workgroup. So we need to inspect our session to get this info.